hollow/src/main/java/com/netflix/hollow/tools/stringifier/HollowRecordJsonStringifier.java

```diff
@@ -384,9 +398,12 @@ private void appendFieldStringify(Writer writer, HollowDataAccess dataAccess, in
     }
 
     private String escapeString(String str) {
-        if(str.indexOf('\\') == -1 && str.indexOf('\"') == -1)
-            return str;
-        return str.replace("\\", "\\\\").replace("\"", "\\\"");
+        for (Character escapeSequence : escapedSymbols.keySet()) {
```

Comment:
I'm a little concerned about the efficiency here. It wasn't great before this change, but now it's 7x worse.

The `#indexOf` should be O(n) given that the pattern to match is 1 char always. But we do it 7 times now.

The `str = str.replace(escapeSequence.toString(), escapedSymbols.get(escapeSequence));` will generate 7 objects to GC. I wonder if we can use `StringBuilder here? If we're able to calculate or estimate the replacements beforehand we can construct one with an initial size that isn't likely to need to grow partway through building it.

For simple, relatively shallow objects this is negligible. But we're already encountering data models with some large object graphs for which this stringify already performs pretty badly.